python/pyspark/mllib/stat/distribution.py

```diff
@@ -15,12 +15,14 @@
 # limitations under the License.
 #
 
-from collections import namedtuple
-
 __all__ = ['MultivariateGaussian']
 
+from typing import NamedTuple
+
+from pyspark.mllib.linalg import Matrix, Vector
+
 
-class MultivariateGaussian(namedtuple('MultivariateGaussian', ['mu', 'sigma'])):
+class MultivariateGaussian(NamedTuple):
```

Comment:
Hm, is't it a breaking change? Previously it inherited an actual named tuple that works like named tuple. Now it inherits the typing class of `NamedTuple`.

Reply:
Please take a look at https://github.com/apache/spark/pull/34375#discussion_r738143060 .
I followed that one. Are they different?

Reply:
Oh, okay, it was my bad. I didn't reliased that the typing annotation can be same https://docs.python.org/3/library/typing.html#typing.NamedTuple. Resolving this.